contracts/token/ERC20Mintable.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity ^0.6.0;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "../access/Community.sol";
+
+
+/**
+ * @dev Extension of {ERC20} that gives the owner permission to mint (create) new tokens as he sees fit.
+ */
+contract ERC20Mintable is ERC20, Community {
```

Comment:
What is the difference between this one and openzeppelin's ERC20Mintable?

Reply:
ERC20Mintable (or ERC721Mintable) is gone from openzeppelin-contracts. They are refactoring them.